hosting-companies.md

```diff
@@ -87,7 +87,8 @@ In alphabetical order:
 * [XEL](https://xel.nl)
 * [Zenbox](https://zenbox.pl)
 * [Zenith Media Canada](https://zenithmedia.ca/wordpress-website-hosting/)
-
+* [True-False Hosting](https://truefalsehosting.com)
+  
```

Comment:
This should be in alphabetical order with the others (so under Templ)

Reply:
Sorry for that. Fixed now.